bytecomp/lambda.ml

```diff
@@ -264,9 +264,25 @@ and lambda_event_kind =
   | Lev_function
   | Lev_pseudo
 
+type module_map = {
+  map :  (int * Ident.t) list;
+  prims : (int * string) list;
+  methcache : int option;
```

Comment:
I don't know about the role of `method_cache` in compilation. Why is it stored here?